README.md

```diff
@@ -82,17 +125,15 @@ the actual local distribution and **not** the outdated cached version.
 
 ### Publishing a new version
 
-Versions are published to PyPi using `twine`.  As a first step, the new package is
-published to the PyPi **test** site, giving a chance to check the package upload before
-committing it to the live PyPi archive.
+Version publication to PyPi also occurs using `poetry`.  As a first step, the new
+package is published to the PyPi **test** site, giving a chance to check the package
+upload before committing it to the live PyPi archive.
```

Comment:
You'll probably need to add notes on configuring credentials for PyPi.

Reply:
Ahh yes, I also need to learn how to configure PyPi credentials 😅